massgov/mayflower
# DP-29849 Accessibility error fix for org. nav

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

1. `li.ma__organization-navigation__item` 
     - Remove `aria-haspopup="true"` from `li.ma__organization-navigation__item`s.

2. `input#organization-navigation-search`
     - Change `type` value to `search`.
     - Add the following attributes:
             - `autocomplete="off"`
             - `inputmode="search"`

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-29849)
- [Github issue]()

## Steps to Test


1. Go to the org nav component in the org page.
2. In inspect, check the markup for the component.
     - `li.ma__organization-navigation__item`s don't have `aria-haspopup="true"`.
     - `input#organization-navigation-search` has  `type="search"`, `autocomplete="off"` and `inputmode="search"`

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
